src/hoard.rs

```diff
@@ -45,9 +45,9 @@ impl Hoard {
     }
 
     pub async fn start(&mut self) -> (String, bool) {
-        if !compare_with_latest_version().await {
+        if !compare_with_latest_version().await.0 {
             println!(
-                "Latest Version Available at https://github.com/Hyde46/hoard \nPlease update."
+                "Running hoard a newer Version {} Available at https://github.com/Hyde46/hoard \nPlease update.", compare_with_latest_version().await.1
```

Comment:
Sounds a little weird now.
`A newer Version ({}) is available at https://github.com/Hyde46/hoard \nPlease update.`